src/control-flow/match-expressions.md

```diff
@@ -21,3 +21,12 @@ expression of the block, if any. In the example above, the type is `()`.
 
 See [pattern matching](../pattern-matching.md) for more details on patterns in
 Rust.
+
+<details>
+    
+* Save the match expression to a variable and print it out.
+* Remove `.as_deref()` and explain the error.
+    * `std::env::Agrs().next` returns an `Option<&String>`, but we cannot match against `String`.
```

Comment:
I guess you mean `Args()` and `next()`.

Reply:
Or rather, `args().next()`, case is important.

Reply:
Follow-up in #400